src/components/SmartSort/SmartSort.tsx

```diff
@@ -0,0 +1,45 @@
+/** @jsx h */
+
+import { h } from 'preact';
+import Template from '../Template/Template';
+import {
+  SmartSortCSSClasses,
+  SmartSortTemplates,
+} from '../../widgets/smart-sort/smart-sort';
+
+type SmartSortProps = {
+  cssClasses: SmartSortCSSClasses;
+  templates: SmartSortTemplates;
+  isSmartSorted: boolean;
+  relevancyStrictness?: number;
+  refine(relevancyStrictness: number | undefined): void;
+};
+
+const SmartSort = ({
+  cssClasses,
+  templates,
+  isSmartSorted,
+  relevancyStrictness,
+  refine,
+}: SmartSortProps) => (
+  <button
```

Comment:
shouldn't it be: 

```
container
  template (text)
  button
    template (button)
```

(let's add this widget to specs too)

Reply:
good suggestion 👍🏼 
https://github.com/algolia/instantsearch.js/pull/4648/commits/3bd77d6216129ef2c4c66054f461f8cc6fb9695e

https://github.com/algolia/instantsearch.js/pull/4648/commits/901fa5d5788bfdec3ae9c33ef1c1bc5992c119d9